test/args_parse.py

```diff
@@ -29,6 +29,7 @@ def parse_common_options(datadir=None,
   parser.add_argument('--fake_data', action='store_true')
   parser.add_argument('--tidy', action='store_true')
   parser.add_argument('--metrics_debug', action='store_true')
+  parser.add_argument('--local_torch_xla_changes', action='store_true')
```

Comment:
I think we should make __version__.py present even in all the setups.
Local dev machines, distributed file, eggs, ...

Reply:
Hmm so you want us to generate `version.py` in the GCE image creation workflow?

What about if users decide to install the wheels themselves and then run, for example, `test_train_mnist.py`? Then they'll see this `ImportError`. I think it's unfeasible to generate this everywhere? The way upstream PyTorch seems to do it is by not overriding the `sys.path` anywhere: [pytorch/setup.py](https://github.com/pytorch/pytorch/blob/master/setup.py#L286).

Reply:
OK, I don't understand. We do generate the version.py always.
In which case this fails?
We do not override sys.path, we insert the path of the development folder in front so that we can test.
In we load from the development torch_xla/ the file is there.
In which case would it be missing?

Reply:
So this file is only generated when `setup.py` is run to install `torch_xla` wheel. Also the GCE image currently is created by installing the wheels but also clones the `pytorch/xla` repository source code in the VM. When cloning the source the `version.py` does not exist.

Similarly if a user were to install the wheels and clone the repo to test our mnist example, version.py would be missing and fail (since the first path in `sys.path` points to the local cloned source instead of the installation and so it'll call into `cloned/path/xla/torch_xla/` in which `version.py` dne.

Reply:
OK, so remove the flag, and inject the folder into sys.path only if version.py exists there.

Reply:
I'm not sure if that's what we want though. This sounds weird to me: Let's say I want to test out some changes on `data_parallel.py` and I've installed `torch_xla` wheel and cloned the `pytorch/xla` github repo. Then basically to test the `data_parallel.py` overriding I'd need to go create `xla/torch_xla/version.py` with specifically the contents: `__version__ = 'abc'`?

Reply:
Yes, if you want to "develop" you need to `python setup.py install`. This is common practice.

Reply:
I've done what you suggested.